<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRemoteDialog</class>
 <widget class="QDialog" name="EditRemoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Mount Config</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hostnameLabel">
     <property name="text">
      <string>&amp;Hostname:</string>
     </property>
     <property name="buddy">
      <cstring>hostnameComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="hostnameComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editConfigButton">
       <property name="icon">
        <iconset theme="gtk-edit"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>&amp;User:</string>
     </property>
     <property name="buddy">
      <cstring>userLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="userLineEdit">
     <property name="placeholderText">
      <string>leave empty for user from config</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="remoteMountpointLabel">
     <property name="text">
      <string>&amp;Remote Mountpoint:</string>
     </property>
     <property name="buddy">
      <cstring>remoteMountpointLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="remoteMountpointLineEdit">
     <property name="placeholderText">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="localMountpointLabel">
     <property name="text">
      <string>&amp;Local Mountpoint:</string>
     </property>
     <property name="buddy">
      <cstring>localMountpointPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPathEdit" name="localMountpointPathEdit">
     <property name="style">
      <enum>QPathEdit::JoinedButton</enum>
     </property>
     <property name="dialogButtonIcon">
      <iconset theme="dialog-object-properties">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="pathMode">
      <enum>QPathEdit::ExistingFolder</enum>
     </property>
     <property name="dialogOptions">
      <set>QFileDialog::DontResolveSymlinks|QFileDialog::ShowDirsOnly</set>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mountLabel">
     <property name="text">
      <string>&amp;Mount:</string>
     </property>
     <property name="buddy">
      <cstring>mountCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="mountCheckBox"/>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPathEdit</class>
   <extends>QWidget</extends>
   <header>qpathedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditRemoteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditRemoteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
